examples/python/rgbd/main.py

```diff
@@ -117,6 +123,8 @@ def download_progress(url: str, dst: Path) -> None:
     From: https://gist.github.com/yanqd0/c13ed29e29432e3cf3e7c38467f42f51
     """
     resp = requests.get(url, stream=True)
+    if resp.status_code != 200:
+        raise ValueError("Failed to download file")
```

Comment:
```suggestion
        raise ValueError(f"Failed to download file: {resp.statuc_code}")
```

adding the message would be nice too